packages/jsii-pacmak/test/expected.jsii-calc-lib/dotnet/Amazon.JSII.Tests.CalculatorPackageId.LibPackageId/.jsii

```diff
@@ -364,6 +444,10 @@
             "summary": "The value."
           },
           "immutable": true,
+          "locationInModule": {
```

Comment:
Wouldn't it be sufficient to use a single line with `:` to separate file/line? Just thinking of how much global energy this will preserve.

Reply:
Aren't you the king of Worse Is Better ;)

Reply:
Always. That's what I strive for. For example, stringified tokens.